shared/js/fb/fb_data_reader.js

```diff
@@ -4,8 +4,8 @@
 // To use this library you need to include 'shared/js/fb/fb_request.js'
 // WARNING: This file lazy loads 'shared/js/fb/fb_tel_index.js'
 
-var fb = this.fb || {};
```

Comment:
I think window.fb may not match the right solution, please check related files to see if this.fb is in context